### ISSUE TITLE
only updates to issue #905, no higher

### ISSUE THREAD

**Issue by veganline** [thirtybees/coreupdater#13]
I installed TB from Softaculous and used **preferences > core updater > bleeding edge**
The latest possible version has been #905, consistently over several different installs.
Maybe I'm missing something obviou, but it's a pity as there are some very useful fixes after #905

![image](https://user-images.githubusercontent.com/3786599/90528213-46dea200-e16a-11ea-983e-027183a0248a.png)

**Comment by getdatakick** [comment]
This is misunderstanding, not a bug.

The versions displayed in the dropdown corresponds to branches that exists on github. Issue branches are usually a short-lived, as they exists only during development of the fix. Once the issue is fixed, the issue-xxx branch is merged into `1.1.x`, and issue branch is deleted.

This means that you should always use only `1.1.x`.

`issue-xxx` branches should be used only by developers, or by users that reported the bug to verify that the issue is indeed fixed (before the fix is included into mainline). The code in these branches are not approved and properly tested. Yet.
